tools/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -33,6 +33,28 @@ object ToolUtils extends Logging {
     df.showString(numRows, 0)
   }
 
+  def parseClusterTags(clusterTag: String): (String, String, String, String) = {
```

Comment:
this should have a description its a very specific function. Can we just return a map here of key -> value and return everything here and select the ones we want later?

Reply:
Done. Code updated to return map of key-> value pair. It will be easy later if we want to print more cluster tags. Only OutputWriter needs to be updated as we have all the data in QualificationSummaryInfo.